docs/en_US/guide/sinks/builtin/mqtt.md

```diff
@@ -14,9 +14,9 @@ The action is used for publish output message into an MQTT server.
 | certificationPath    | true     | The certification path. It can be an absolute path, or a relative path. If it is an relative path, then the base path is where you excuting the `kuiperd` command. For example, if you run `bin/kuiperd` from `/var/kuiper`, then the base path is `/var/kuiper`; If you run `./kuiperd` from `/var/kuiper/bin`, then the base path is `/var/kuiper/bin`. |
 | privateKeyPath       | true     | The private key path. It can be either absolute path, or relative path, which is similar to use of certificationPath.                                                                                                                                                                                                                                     |
 | rootCaPath           | true     | The location of root ca path. It can be an absolute path, or a relative path, which is similar to use of certificationPath.                                                                                                                                                                                                                               |
-| rawCert            | true | base64 encoded original text of cert |
-| rawKey             | true | base64 encoded original text of key |
-| rawCA              | true | base64 encoded original text of CA |
+| rawCertification            | true | base64 encoded original text of cert, use `certificationPath` first if both defined |
```

Comment:
Is it better to align with the existing naming pattern? By this way, when ordering by alphabet, the two property can be side by side: certificationRaw and certiricationPath.